libs/portal-integration-angular/src/lib/core/directives/loading-indicator.directive.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ComponentFactoryResolver,
+  ComponentRef,
+  Directive,
+  ElementRef,
+  Input,
+  OnChanges,
+  Renderer2,
+  SimpleChanges,
+  ViewContainerRef,
+} from '@angular/core';
+import { LoadingIndicatorComponent } from '../components/loading-indicator/loading-indicator.component';
+
+@Directive({
+  selector: '[onecxLoadingIndicator]',
```

Comment:
rename to `'[ocxLoadingIndicator]'`, because there is a naming convention that it should start with 'ocx...'